README.md

```diff
@@ -176,6 +164,105 @@ class AwesomeSpiderWithPage(scrapy.Spider):
   Scrapy request workflow (Scheduler, Middlewares, etc).
 
 
+## Multiple browser contexts
+
+Multiple [browser contexts](https://playwright.dev/python/docs/core-concepts/#browser-contexts)
+to be launched at startup can be defined via the `PLAYWRIGHT_CONTEXTS` [setting](#settings).
+
+### Choosing a specific context for a request
+
+Pass the name of the desired context in the `playwright_context` meta key:
+
+```python
+yield scrapy.Request(
+    url="https://example.org",
+    meta={"playwright": True, "playwright_context": "first"},
+)
+```
+
+### Creating a context during a crawl
+
+If the context specified in the `playwright_context` meta key does not exist, it will be created.
+You can specify keyword arguments to be passed to [`Browser.new_context`](https://playwright.dev/python/docs/api/class-browser#browsernew_contextkwargs)
+in the `playwright_context_kwargs` meta key:
+
+```python
+yield scrapy.Request(
+    url="https://example.org",
+    meta={
+        "playwright": True,
+        "playwright_context": "new",
+        "playwright_context_kwargs": {
+            "java_script_enabled": False,
+            "ignore_https_errors": True,
+            "proxy": {
+                "server": "http://myproxy.com:3128",
+                "username": "user",
+                "password": "pass",
+            },
+        },
+    },
+)
+```
```

Comment:
What happens if a context already exists with the same name?

Reply:
The existing context is used in that case, and `playwright_context_kwargs` are ignored.

Reply:
I just added a note to the readme about ignoring the kwargs.